pkg/proc/eval.go

```diff
@@ -67,16 +67,18 @@ func ConvertEvalScope(dbp *Target, gid, frame, deferCall int) (*EvalScope, error
 	if err != nil {
 		return nil, err
 	}
-	if g == nil {
-		return ThreadScope(ct)
-	}
 
 	var opts StacktraceOptions
 	if deferCall > 0 {
 		opts = StacktraceReadDefers
 	}
 
-	locs, err := g.Stacktrace(frame+1, opts)
+	var locs []Stackframe
```

Comment:
I'm guessing that the reason for this change is that there is a bug where the frame parameter is ignored if we can't find the requested goroutine. I don't think we should sneak this bug fix in here without any comment.
It should be in a separate PR, if possible with a test in pkg/proc/proc_test.go.

Reply:
Done in #2563